ajax/libs/print-js/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "print-js",
+  "filename": "print.min.js",
```

```suggestion
  "filename": "print.js",
```
Only `print.js` is included in the package, not `print.min.js`.

---

updated